scripts/clang-format/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cd arangodb
+#find arangod -regex '.*\.\(cpp\|hpp\|cu\|c\|h\)' -exec clang-format -style=file -i {} \;
+find arangosh -regex '.*\.\(cpp\|hpp\|cu\|c\|h\)' -exec clang-format -style=file -i {} \;
```

Comment:
IMO we don't need to care about `.cu` files here. A common file extension for C++ files is also `.cc`, which is not in the list (and also not used by us). We may or may not want to add this to be "future-proof" :sweat_smile: